py4DSTEM/io/datastructure/py4dstem/io.py

```diff
@@ -42,14 +42,11 @@ def Calibration_from_Metadata(metadata):
         (Calibration)
     """
     from py4DSTEM.io.datastructure.py4dstem.calibration import Calibration
-    p = metadata._params
-    metadata.__class__ = Calibration
-    metadata.__init__(
-        name = metadata.name
-    )
-    metadata._params.update(p)
+    
+    cal = Calibration(name = metadata.name)
```

Comment:
This should fix the calibration problems, where `_targets` was missing, but I want a test of that before merging